deployment/build.gradle

```diff
@@ -66,11 +64,11 @@ buildDeb {
 }
 
 docker {
-    name "trellisldp/trellis-ext-db:${trellisVersion}"
+    name "trellisldp/trellis-ext-db:${project.version}"
     tags 'latest'
     dockerfile file('src/docker/Dockerfile')
-    files tasks.distTar.outputs, 'deployment/src/dist/etc/config.yml','deployment/src/docker/command.sh'
-    buildArgs([BUILD_VERSION: trellisVersion])
+    files tasks.distTar.outputs, './src/dist/etc/config.yml','./src/docker/command.sh'
+    buildArgs([BUILD_VERSION: project.version])
     def gitLabels = ['git.url':'https://github.com/trellis-ldp/trellis-ext-db']
```

Comment:
This code doesn't capture the git labels properly for me. I had better luck with this:
```
try {
    def git = grgit.open()
    describe = git.head().abbreviatedId
    url = git.remote.list().find({ it.name == 'origin' }).url
} catch (all) {
    println(all)
}
labels(['git.commit.id.describe': describe, 'git.remote.origin.url': url])
```

Reply:
Thanks, I'll update the code to use this.